src/main/resources/templates/springboot/rest/application.properties.tmpl

```diff
@@ -1,11 +1,5 @@
-spring.jersey.type=servlet
-server.servlet-path=/admin
 server.port=-1
 
-# disable security for verbose /health output
-endpoint.health.sensitive=false
-management.security.enabled=false
```

Comment:
these aren't required for mvc projects? i can update our teams projects...

Reply:
yeah; previously, we used nginx to limit incoming calls to only those defined in swagger. but with app services that's no longer possible, so until we pivot away from that crappy solution, we need to secure all endpoints.